ietf-document/draft-quic-4-sat/draft-kuhn-quic-4-sat-04.txt

```diff
@@ -0,0 +1,728 @@
+   time reduction in a SATCOM use-case [ICCRG100].
+
+   [NCT13] and [RFC3135] describe the main functions of SATCOM TCP split
+   solutions.  Shortly, for traffic originated at a gateway to an
+   endpoint connected via a satellite terminal, the TCP split intercepts
+   TCP SYN packets to act on behalf of the endpoint and adapt the data
+   rate transmission to the SATCOM scenario.  The split solution
+   specifically tunes the TCP parameters to the context (latency,
+   available capacity) of each link.  When a PEP is used on each side of
+   the satellite link, a protocol other than TCP, optimized for the
+   satellite link, may be used.  The tuning can be achieved using a
+   priori information about the satellite system and/or by measuring the
+   properties of the network segment that includes the satellite system.
+   Split connections also allow for recovery from packet losses that is
+   local to the part of the connection on which the packet losses
+   occurred.  This eliminates the need for end-to-end recovery of lost
+   packets.
+
+   One important advantage of a TCP split solution is that it does not
+   require any end-to-end modifications and is independent for both
+   client and server sides.  That being said, this comes with a
+   drawback: TCP splitters often are unable to track the most recent
+   end-to-end improvements in protocol mechanisms (e.g., RACK, ECN, TCP
+   Fast Open support) contributing to ossification of the transport
+   system.  The methods configured in the split proxy usually continue
+   to be used until a split solution is finally updated.  This can
+   delay/negate the benefit of any end-to-end improvements.
+
+4.  Regression tests
+
+   This section proposes regression tests for QUIC.  We define by:
+
+   o  Download path: from Internet to host
+
+   o  Upload path: from the host to Internet
+
+
+
+
+Kuhn, et al.             Expires August 31, 2020                [Page 5]
+
+Internet-Draft                 QUIC 4 SAT                  February 2020
+
+
+4.1.  Small public satellite broadband access
+
+   The characteristics of the architecture under test are the following:
+
+   o  Satellite downlink path: 10 Mbps
+
+   o  Satellite uplink path: 2 Mbps
+
+   o  No emulated packet loss
+
+   o  RTT: 650 ms
+
+   o  Buffer size : BDP
+
+   During the transmission of 100 MB on both download and upload paths,
+   the test should report the downloading time of 2 MB, 10 MB and 100
+   MB.
+
+   Initial thoughts of the performance obectives for QUIC are the
+   following:
+
+   o  3 s for downloading 2 MB
+
+   o  10 s for downloading 10 MB
+
+   o  85 s for downloading 100 MB
+
+   o  10 s for uploading 2 MB
+
+   o  50 s for uploading 10 MB
+
+   o  420 s for uploading 100 MB
+
+4.2.  Medium public satellite broadband access
+
+   The characteristics of the architecture under test are the following:
+
+   o  Satellite downlink path: 50 Mbps
+
+   o  Satellite uplink path: 10 Mbps
+
```

Comment:
Do we insert /link/ before /packet loss/ - to be sure we do not mean no buffer overflow.